students/conferences.md

```diff
@@ -83,17 +83,17 @@ We are not able to support our students with travel and accommodation. If you ha
 
 Interested? Any further questions? Write us an <a href="mailto:contact@rgsoc.org">email</a> and we can talk about the details.
 
-<h3 id="conferences">Conference Sponsors in 2017</h3>
+<h3 id="conferences">Conference Sponsors in 2018</h3>
 
-A huge **thank you** to all conferences who supported our students and provided free tickets during Rails Girls Summer of Code 2017.
+A huge **thank you** to all conferences who supported our students and provided free tickets during Rails Girls Summer of Code 2018.
 
 <section class="conferences-container">
 
   {% for conference in site.data.conferences %}
     <div class="conference-item">
       <p class="conference-logo">
         <a href="{{conference.url}}">
-        <img src="{{conference.img}}"></a>
+        <img width="200" height="200" src="{{conference.img}}"></a>
```

Comment:
could we move this styling bit into the css? maybe into `layout.scss` for now until we refactor/clean up those files 😇

Reply:
Thanks for the feedback! 

Hmm, I'm not sure, but I don't think this is styling but image properties... not sure we can extract these into CSS. 

I could be wrong. Will have a look

Reply:
Are all images resized ahead of time to have exactly those sizes? (sorry, the PR wasn't showing me the conference logos) — if that's the case, then yes, you're right and it's the image properties that need to be defined in the html tag. If we need to “resize” the image (i.e. if it's not exactly the size we want it to be) then that would fall under styling and need to be moved to the css. I thought we were dealing with the latter. Could you confirm that's not the case? If so, we can go ahead and merge this!

Reply:
AH! Yes, good point, thanks @alicetragedy. I should be doing the latter. On it!